experimental/regression_suite/shark-test-suite-models/sd3/test_clip.py

```diff
@@ -0,0 +1,167 @@
+    "--iree-llvmcpu-distribution-size=32",
+    "--iree-opt-const-eval=false",
+    "--iree-llvmcpu-enable-ukernels=all",
+    "--iree-global-opt-enable-quantized-matmul-reassociation",
+]
+
+
+@pytest.fixture
+def SD3_CLIP_COMMON_RUN_FLAGS(
+    sd3_clip_inference_input_0,
+    sd3_clip_inference_input_1,
+    sd3_clip_inference_input_2,
+    sd3_clip_inference_input_3,
+    sd3_clip_inference_input_4,
+    sd3_clip_inference_input_5,
+    sd3_clip_inference_output_0,
+    sd3_clip_inference_output_1,
+):
+    return [
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_0.path}",
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_1.path}",
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_2.path}",
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_3.path}",
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_4.path}",
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_5.path}",
+        f"--expected_output=2x154x4096xf32=@{sd3_clip_inference_output_0.path}",
+        f"--expected_output=2x2048xf32=@{sd3_clip_inference_output_1.path}",
+    ]
+
+
+ROCM_COMPILE_FLAGS = [
+    "--iree-hal-target-backends=rocm",
+    f"--iree-rocm-target-chip={rocm_chip}",
+    "--iree-input-type=torch",
+    "--iree-opt-const-eval=false",
+    "--iree-global-opt-propagate-transposes=true",
+    "--iree-opt-outer-dim-concat=true",
+    "--iree-rocm-waves-per-eu=2",
+    "--iree-llvmgpu-enable-prefetch",
+    "--iree-flow-enable-aggressive-fusion",
+    "--iree-global-opt-enable-fuse-horizontal-contractions=true",
+    "--iree-opt-aggressively-propagate-transposes=true",
+    "--iree-codegen-llvmgpu-use-vector-distribution=true",
+    "--iree-execution-model=async-external",
+    "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline, util.func(iree-preprocessing-pad-to-intrinsics{pad-target-type=conv}))",
+]
+
+###############################################################################
+# CPU
+###############################################################################
+
+
+def test_compile_clip_cpu(sd3_clip_mlir):
+    VmfbManager.sd3_clip_cpu_vmfb = iree_compile(
+        sd3_clip_mlir, "cpu", CPU_COMPILE_FLAGS
+    )
+
+
+@pytest.mark.depends(on=["test_compile_clip_cpu"])
+def test_run_clip_cpu(SD3_CLIP_COMMON_RUN_FLAGS, sd3_clip_real_weights):
+    iree_run_module(
+        VmfbManager.sd3_clip_cpu_vmfb,
+        device="local-task",
+        function="encode_tokens",
+        args=[
+            f"--parameters=model={sd3_clip_real_weights.path}",
+            "--expected_f32_threshold=0.15f",
+        ]
+        + SD3_CLIP_COMMON_RUN_FLAGS,
+    )
+
+
+###############################################################################
+# ROCM
+###############################################################################
+
+
+@pytest.mark.xfail(
+    strict=True,
+    reason="Expected compilation to fail (remove xfail for test_compile_clip_rocm)",
```

Comment:
I'd drop the `(remove xfail` part of this reason. I put similar messages in the generated ONNX tests to try making it easier to understand xfails but I think I failed there :P. I changed the language a bit in https://github.com/nod-ai/SHARK-TestSuite/pull/277/files#diff-cd2c3d64b04bf814e45e0b757ffa1c1625ed0b5371b22a4892131b76336a00c5 too, from `Expected run to fail (remove from 'expected_run_failures')` to `Expected run to fail (included in 'expected_run_failures')`

https://docs.pytest.org/en/stable/how-to/skipping.html#reason-parameter suggests something like "known compile failure"

This text is also cut off for using too many characters:
```
Compiling /mnt/share/amdsharks/iree_tests_cache/artifacts/sd3_clip/model.mlirbc -> /mnt/share/amdsharks/iree_tests_cache/artifacts/sd3_clip/model.rocm_gfx90a.vmfb with flags:
  --iree-hal-target-backends=rocm
  --iree-rocm-target-chip=gfx90a
  --iree-input-type=torch
  --iree-opt-const-eval=false
  --iree-global-opt-propagate-transposes=true
  --iree-opt-outer-dim-concat=true
  --iree-rocm-waves-per-eu=2
  --iree-llvmgpu-enable-prefetch
  --iree-flow-enable-aggressive-fusion
  --iree-global-opt-enable-fuse-horizontal-contractions=true
  --iree-opt-aggressively-propagate-transposes=true
  --iree-codegen-llvmgpu-use-vector-distribution=true
  --iree-execution-model=async-external
  --iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline, util.func(iree-preprocessing-pad-to-intrinsics{pad-target-type=conv}))
XFAIL (Expected compilation to fail (remove xfail for test_compile_c...)
experimental/regression_suite/shark-test-suite-models/sd3/test_mmdit.py::test_compile_mmdit_rocm 
```

https://github.com/iree-org/iree/actions/runs/9913116418/job/27389778991?pr=17883#step:10:54

Reply:
Ah lol, not very useful I guess